playground.txt

```diff
@@ -16,6 +16,30 @@ You are now connected to database "nc_news_test" as user "jakubwiszowaty".
          13 | Another article about Mitch                            | mitch | butter_bridge | There will never be enough articles about Mitch!                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | 2020-10-11 12:24:00 |     0 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700
 (13 rows)
 
+<<<<<<< HEAD
```

Comment:
This playground.txt and .sql file shouldn't be included in the PR

Reply:
I hope I've managed to remove it, but I can't see it in my VS code :/